plugins/HHAnalyzer.cc

```diff
@@ -195,6 +242,11 @@ void HHAnalyzer::analyze(const edm::Event& event, const edm::EventSetup&, const
             else {
                fillTriggerEfficiencies(leptons[ilep1], leptons[ilep2], dilep);
             }
+
+            if (!dilep.isOS)
+                continue;
+            if (event.isRealData() && !(leptons[ilep1].hlt_DR_matchedObject < 0.3 && leptons[ilep1].hlt_DR_matchedObject < 0.3))
```

Comment:
Ok, the cut can be set at 0.1 (see plot below for 2015D DoubleElectron showing the DR with matched object). Actually there is a parameter in the config https://github.com/cp3-llbb/HHAnalysis/blob/master/test/HHConfiguration.py#L50 which specifies the cut.  
![drmatchedobject](https://cloud.githubusercontent.com/assets/3646269/12698325/5852aa16-c79a-11e5-8719-30a9bc213630.png)